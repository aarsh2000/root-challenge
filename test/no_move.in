Driver A1
Driver Bob
Driver -
Trip A1 07:15 07:15 17.3
Trip Bob 04:15 04:15 0
Trip - 10:01 13:16 0
