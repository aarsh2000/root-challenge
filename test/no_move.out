-: 0 miles
Bob: 0 miles
A1: 0 miles
